### ISSUE TITLE
Unthemed  parts

### ISSUE THREAD

**Issue by EliaHKM** [dracula/krita#2]
Hi,
Some parts and shadows are in blue color instead of purple 
[https://imgur.com/a/aMfWuIu](url)
thanks in advance
have a nice day
peace

**Comment by ghost** [comment]
Yeah this theme is just straight up unfinished & never got updated :P
